packages/blocks/drawing-canvas/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@hashintel/drawing-canvas",
+  "version": "0.0.0",
+  "description": "A drawing block that leverages tldraw",
+  "keywords": [
+    "blockprotocol",
+    "blocks"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/hashintel/hash.git#vu/drawing-canvas",
+    "directory": "packages/blocks/drawing-canvas"
+  },
+  "license": "MIT",
+  "author": "teenoh",
+  "scripts": {
+    "build": "block-scripts build",
+    "dev": "block-scripts dev",
+    "fix:eslint": "eslint --ext .ts,.tsx --fix ./src/",
+    "lint:eslint": "eslint --ext .ts,.tsx ./src/",
+    "lint:tsc": "tsc --noEmit",
+    "serve": "block-scripts serve"
+  },
+  "dependencies": {
+    "@tldraw/tldraw": "1.11.0",
+    "blockprotocol": "0.0.8",
+    "lodash": "4.17.21",
+    "mobx": "6.5.0",
```

Comment:
I could not find `mobx` and `lodash` in block code. Can we remove them?

Reply:
Thanks for catching. I have removed `lodash` but `mobx` is a dependency `tldraw` needs in order to work. I have moved it to the `devDependencies` list. 
Updates in => https://github.com/hashintel/hash/pull/515/commits/a1b282688f34b2b69775aefc836fe2c735d1f689

Reply:
Hmm interesting. If `lodash` and `mobx` are not listed as deps, how would the block work in prod then? Not sure why `@tldraw/tldraw` does not mention them as `peerDependencies` 🤔
https://unpkg.com/browse/@tldraw/tldraw@1.11.1/package.json

Reply:
Ohh lodash isn't a dependency. I added that when I initially needed one of its methods, but removed that in the commit I mentioned. Not sure why `mobx` isn't listed as a peerDependency either.